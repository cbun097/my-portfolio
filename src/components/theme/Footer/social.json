[
  {
    "id": 0,
    "name": "Telegram",
    "link": "https://t.me/cbun097",
    "icon": "/icons/telegram.svg"
  },
  {
    "id": 1,
    "name": "Github",
    "link": "https://github.com/cbun097",
    "icon": "/icons/github.svg"
  },
  {
    "id": 3,
    "name": "Twitter",
    "link": "https://twitter.com/cbun097",
    "icon": "/icons/twitter.svg"
  },
  {
    "id": 4,
    "name": "YouTube",
    "link": "https://www.youtube.com/theClairebmusic",
    "icon": "/icons/youtube.svg"
  },
  {
    "id": 5,
    "name": "Linkedin",
    "link": "https://www.linkedin.com/in/claire-bun-3678ba132/",
    "icon": "/icons/linkedin.svg"
  },
  {
    "id": 6,
    "name": "Instagram",
    "link": "https://www.instagram.com/cbun097/",
    "icon": "/icons/instagram.svg"
  }
]
