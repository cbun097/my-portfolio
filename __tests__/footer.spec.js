import React from 'react'
import renderer from 'react-test-renderer'
import { Footer } from '../src/components/theme/Footer/index'

describe('Footer Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Footer />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
