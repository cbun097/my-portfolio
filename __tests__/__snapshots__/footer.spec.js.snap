// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component renders correctly 1`] = `
<div
  className="sc-fznyAO dkgGhB"
>
  <div
    className="sc-Axmtr irmYez sc-fznKkj cNTHwD"
  >
    <div
      className="sc-fzokOt hKbVUM"
    >
      <h2>
        Claire Bun
      </h2>
      <span>
        Â© All rights are reserved | 
        2020
         | Made with
         
        <span
          aria-label="love"
          role="img"
        >
          ðŸ’–
        </span>
         
        by
         
        <a
          href="https://smakosh.com/?ref=portfolio-dev"
          rel="noopener noreferrer"
          target="_blank"
        >
          Smakosh
        </a>
      </span>
    </div>
    <div
      className="sc-fznZeY fEioGA"
    >
      <a
        aria-label="follow me on Telegram"
        href="https://t.me/cbun097"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Telegram"
          src="/icons/telegram.svg"
          width="24"
        />
      </a>
      <a
        aria-label="follow me on Github"
        href="https://github.com/cbun097"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Github"
          src="/icons/github.svg"
          width="24"
        />
      </a>
      <a
        aria-label="follow me on Twitter"
        href="https://twitter.com/cbun097"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Twitter"
          src="/icons/twitter.svg"
          width="24"
        />
      </a>
      <a
        aria-label="follow me on YouTube"
        href="https://www.youtube.com/theClairebmusic"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="YouTube"
          src="/icons/youtube.svg"
          width="24"
        />
      </a>
      <a
        aria-label="follow me on Linkedin"
        href="https://www.linkedin.com/in/claire-bun-3678ba132/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Linkedin"
          src="/icons/linkedin.svg"
          width="24"
        />
      </a>
      <a
        aria-label="follow me on Instagram"
        href="https://www.instagram.com/cbun097/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Instagram"
          src="/icons/instagram.svg"
          width="24"
        />
      </a>
      <a
        aria-label="follow me on Dev.to"
        href="https://dev.to/cbun097"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Dev.to"
          src="/icons/dev-brands.svg"
          width="24"
        />
      </a>
    </div>
  </div>
</div>
`;
